### ISSUE TITLE
official/Recommendation/ncf_keras_main.py: Evaluation Loss=0!

### ISSUE THREAD

**Issue by Beheshtian** [tensorflow/models#7635]
### System information
- **What is the top-level directory of the model you are using**: tensorflow/models
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Linux Ubuntu 18.04
- **TensorFlow installed from (source or binary)**: pip3 install tensorflow-gpu
- **TensorFlow version (use command below)**: 2.0.0
- **Bazel version (if compiling from source)**: No
- **CUDA/cuDNN version**: cuda 10.0 / cudnn 7
- **GPU model and memory**:  Geforce GT 710 / 2gb
- **Exact command to reproduce**: run ncf_keras_main.py with python 3.6.8

### Describe the problem
Dear Experts
I do not change the code in the repository but when i run ncf_keras_main.py , the evaluation loss is zero! 
Could you help me to solve the problem?
Regards

### Source code / logs

Source code: ncf_keras_main.py

Logs:
INFO:tensorflow:BenchmarkMetric: {'global step':9900, 'time_taken': 0.577315,'examples_per_second': 173215.562997}
I1008 12:14:20.484212 140177274066752 keras_utils.py:85] BenchmarkMetric: {'global step':9900, 'time_taken': 0.577315,'examples_per_second': 173215.562997}
2019-10-08 12:14:20.742895: W tensorflow/core/common_runtime/base_collective_executor.cc:216] BaseCollectiveExecutor::StartAbort Out of range: End of sequence
	 [[{{node IteratorGetNext}}]]
INFO:tensorflow:BenchmarkMetric: {'epoch':1, 'time_taken': 46.069036}
4971/4971 - 46s - loss: 0.2801 - hr_sum: 0.0000e+00 - hr_count: 0.0000e+00 - val_loss: 0.0000e+00 - val_hr_sum: 0.1616 - val_hr_count: 1.0000
I1008 12:14:38.118526 140177274066752 keras_utils.py:93] BenchmarkMetric: {'epoch':1, 'time_taken': 46.069036}
I1008 12:14:38.121345 140177274066752 ncf_keras_main.py:309] Training done. Start evaluating
6040/6040 - 17s - loss: 0.0000e+00 - hr_sum: 0.1616 - hr_count: 1.0000
I1008 12:14:54.677765 140177274066752 ncf_keras_main.py:314] Keras evaluation is done.

**Comment by littleV** [comment]
Hi,

I'm having the same issue!

**Comment by pasikon** [comment]
Hey, same here!

**Comment by jvishnuvardhan** [comment]
@Beheshtian This is a stale issue.  We are checking to see if you still need help on this issue. Can you please test the issue with the latest TensorFlow (`TF2.9.1` and `tf-nightly`) and let us know If the issue still persists with the recent versions of TF. Thanks!
